workspaces/spectacle/src/spectacle.ts

```diff
@@ -232,6 +237,51 @@ export async function makeSpectacle(opticContext: IOpticContext) {
         );
         return Promise.resolve(endpointChanges);
       },
+      field: (
+        parent: any,
+        { fieldId }: { fieldId: string },
+        context: GraphQLContext
+      ) => {
+        const fieldNodeWrapper = context
+          .spectacleContext()
+          .shapeQueries.findNodeById(fieldId);
+
+        if (
+          !fieldNodeWrapper ||
+          fieldNodeWrapper.result.type !== shapes.NodeType.Field
+        )
+          return Promise.resolve(null);
+
+        let fieldNode = fieldNodeWrapper.result;
+
+        let shapeNodeWrapper = context
+          .spectacleContext()
+          .shapeQueries.listIncomingNeighborsByType(
+            fieldId,
+            shapes.NodeType.Shape
+          )
+          .results.find((shapeNode) => {
+            return shapeNode.result.type === shapes.NodeType.Shape;
+          });
+
+        if (
+          !shapeNodeWrapper ||
+          shapeNodeWrapper.result.type !== shapes.NodeType.Shape
+        ) {
+          throw new Error('field should describe a shape');
+        }
+
+        let shapeNode = shapeNodeWrapper.result;
+
+        return Promise.resolve({
```

Comment:
I think field is the only node we don't have as a Wrapper node (because we use the spectacleShapeViewerProjection)